rerun_cpp/README.md

```diff
@@ -1,17 +1,10 @@
 # Rerun C++ SDK
 
-This is not yet ready to be used.
+Read our [getting-started guide](https://www.rerun.io/docs/getting-started/cpp?speculative-link) for information on how to use the Rerun C++ SDK.
 
-## Requirements
+## Development
+### Requirements
 Run `scripts/setup.sh`.
 
-## Test it
+### Test it
```

Comment:
Shouldnt that say `just cpp-test` instead? We'll want to remove those shell scripts pretty soon...